src/waveform/renderers/waveformrendermark.cpp

```diff
@@ -40,27 +40,12 @@ void WaveformRenderMark::setup(const QDomNode& node, const SkinContext& context)
 
 void WaveformRenderMark::draw(QPainter* painter, QPaintEvent* /*event*/) {
     PainterScope PainterScope(painter);
-    // Maps mark objects to their positions in the widget.
-    QMap<WaveformMarkPointer, int> marksOnScreen;
-    /*
-    //DEBUG
-    for (int i = 0; i < m_markPoints.size(); i++) {
-        if (m_waveformWidget->getTrackSamples())
-            painter->drawText(40*i,12+12*(i%3),QString::number(m_markPoints[i]->get() / (double)m_waveformWidget->getTrackSamples()));
-    }
-    */
+    // Associates mark objects with their positions in the widget.
+    QList<QPair<WaveformMarkPointer, int>> marksOnScreen;
```

Comment:
alternatively, since `QList<QPair<WaveformMarkPointer, int>>` also shows up in the API, just use a struct instead of a pair? In this case, the `int` doesn't say much so labelling it would make sense IMO.

Reply:
Sure, will do. Even simpler would be to add the int to WaveformMark (or use the existing member m_line position for that), but also here: I wanted to do a minimum refactoring, as close as possible to the original source, not to introduce risk.

Reply:
done